commit_generated.h

```diff
@@ -0,0 +1 @@
+const char *commit="ef4ec68";
```

Comment:
Youre are right: it should not be.  I forgot to add this file to `.gitignore`.  If you pull down the latest version of clnn, it should be excluded by .gitignore.